opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -695,12 +707,15 @@ def zip_model(
             )
 
         # Create a ZipFile Object
-        with ZipFile(zip_file_path, "w") as zipObj:
-            zipObj.write(model_path, zip_file_name_without_extension + "/" + model_name)
+        with ZipFile(str(zip_file_path), "w") as zipObj:
+            zipObj.write(model_path, arcname=str(model_name))
```

Comment:
Could you please explain why you needed to make this change? What was happening before with `zip_file_name_without_extension`?

Reply:
If we use `zip_file_name_without_extension + "/" + model_name`, the zipped file will consist of files like `all-distilroberta-v1/all-distilroberta-v1.pt` and `all-distilroberta-v1/tokenizer.json` in the zip file name `all-distilroberta-v1.zip`.  But I think we want it to be `all-distilroberta-v1.pt` and `tokenizer.json` in `all-distilroberta-v1.zip` similar to how `save_as_pt` and `save_as_onnx` zip the files.